GoToProject/src/components/Reactions/index.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {
+  Cart,
+  CartInfo,
+  CartImage,
+  InfoTitle,
+  TitleCart,
+  TitleDescription,
+  TouchableOpacity,
+  Starts,
+  TextStart,
+} from '../../containers/Reactions/styled';
+import {Image} from 'react-native';
+import {InfoTitleCart} from './InfoTitle';
+import Icon from 'react-native-ionicons';
+import {colors} from '../../library/constants/colors';
+export const ReactionList = ({url, name, appraisal, desciption}) => {
+  return (
+    <Cart h="124px" w="100%" bg={colors.darkGray} dir="row" jc="space-between">
+      <CartImage w="40%" mt="10px">
+        <Image
+          style={{
+            width: 140,
+            height: 124,
+            borderTopLeftRadius: 5,
+            borderBottomLeftRadius: 5,
+          }}
```

Comment:
These styles can be imported from your style file.

Reply:
done